src/main/webapp/jsp/instructorCourseStudentDetails.jsp

```diff
@@ -8,7 +8,7 @@
 <c:set var="jsIncludes">
   <script type="text/javascript" src="/js/instructorCourseStudentDetails.js"></script>
 </c:set>
-<ti:instructorPage title="Student Details" jsIncludes="${jsIncludes}">
+<ti:instructorPage title="${data.studentInfoTable.name}" jsIncludes="${jsIncludes}">
```

Comment:
Is it possible to change this to `${fn:escapeXml(data.studentInfoTable.name)}` ? This should solve the problem i guess

Reply:
yes i think that's the issue. Thanks for pointing out